cluster/addons/fluentd-elasticsearch/es-image/elasticsearch.yml

```diff
@@ -1,8 +1,385 @@
-# These are the Kubernetes-specific settings that differ
-# from the default values.
-cluster.name: kubernetes_logging
-
-path.data: /data/data
-path.work: /data/work
-path.logs: /data/logs
-path.plugins: /data/plugins
\ No newline at end of file
+##################### Elasticsearch Configuration Example #####################
```

Comment:
Didn't we intentionally not pull in this entire commented out file in a recent PR? Why pull it in now?